App/Sagas/NodeLifecycle.ts

```diff
@@ -95,10 +100,19 @@ export function * handleCreateNodeRequest (dispatch: Dispatch) {
 function * createAndStartNode(dispatch: Dispatch): any {
   try {
     yield put(TextileNodeActions.creatingNode())
+    const repoPathExists: boolean = yield call(RNFS.exists, REPO_PATH)
+    if (!repoPathExists) {
+      yield call(RNFS.mkdir, REPO_PATH)
+      yield call(moveTextileFiles)
```

Comment:
If this is a previous installation, we need to move the textile-go files from their old location to the new location.